multicore_runtime/cloud_logging_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+import mock
+import os
+import unittest
+
+import cloud_logging
```

Comment:
Also, currently this code lives inside the same package as the API library, but it is a medium-term plan of the runtimes team to spin it off so that the API library can be packaged separately. Using relative imports will facilitate this (which is one reason typical open-source projects use them).